pkg/kp/rcstore/consul_store.go

```diff
@@ -66,59 +91,42 @@ func (s *consulStore) Create(manifest pods.Manifest, nodeSelector labels.Selecto
 		Disabled:        false,
 	}
 
-	err = s.forEachLabel(rc, func(id, k, v string) error {
-		return s.applicator.SetLabel(labels.RC, id, k, v)
-	})
-
-	// TODO: If the `put` operations fail, we have already labeled the RC,
-	// yet the RC will not exist in the backing Consul KV store.
-	// The labels may have to be cleaned up.
-
-	err = s.put(id, []kvPair{
```

Comment:
made redundant, keys are now marshaled into a single JSON object. see changes in fields.go